gadget/gadget_test.go

```diff
@@ -3920,6 +3920,15 @@ func (s *gadgetYamlTestSuite) TestSaveLoadDiskVolumeDeviceTraits(c *C) {
 	c.Assert(err, IsNil)
 
 	c.Assert(m4, DeepEquals, expPiLUKSMap)
+
+	// if disk-mapping.jso is empty file (zero size), we should handle this as device traits are not
```

Comment:
```suggestion
	// if disk-mapping.json is empty file (zero size), we should handle this as device traits are not
```

Reply:
While this is technically correct this PR is not the place to fix it